client/app/states/login/login.html

```diff
@@ -34,7 +34,7 @@ <h1 id="brand" ng-bind-html="vm.text.brand"></h1><!--/#brand-->
             </div>
 
             <div class="col-xs-4 col-sm-4 col-md-4 pull-right submit">
-              <button type="submit" class="btn btn-primary btn-lg" tabindex="4" translate>Log In</button>
+              <button type="submit" class="btn btn-primary btn-lg" id="loginButton" tabindex="4" translate>Log In</button>
```

Comment:
looks like the id was added just for the test, but it servers no other purpose, in efforts to keep it clean/un-confusing any way you can find the button by the type or innerHtml? (`submit` is unique to this element)